src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/expressions/brgemm_copy_b_buffer_expressions.cpp

```diff
@@ -88,13 +88,9 @@ void CompensationsBufferExpression::init_allocation_size(
     // Compensations are computed during repacking, so we need to round-up allocation shape according to m_inner_n_block
     // because of OneDNN implementation nuances (as in get_repacking_buffer_size).
     // However, the compensations are computed by N dimension, so K dimension doesn't affect the compensations buffer
+    const auto& precision = parent_expr->get_node()->get_input_element_type(0);
     const size_t n_blk = *ov::snippets::utils::get_projected_subtensor(parent_expr->get_input_port(0)).rbegin();
-    if (snippets::utils::is_dynamic_value(n_blk)) {
-        m_allocation_size = snippets::utils::get_dynamic_value<size_t>();
-    } else {
-        const auto& precision = parent_expr->get_node()->get_input_element_type(0);
-        m_allocation_size = std::max(n_blk, compute_inner_n_block(precision));
-    }
+    m_allocation_size = compute_repacked_n_dim(n_blk, precision);
```

Comment:
It's nice that we now reuse the dynamic values handling from `ov::snippets::utils::rnd_up`, and don't have to replicate this logic anywhere else.